data/nmstate/001-rbac.yaml

```diff
@@ -113,9 +123,16 @@ rules:
   - nodes
   - namespaces
   verbs:
-  - get
+  - '*'
```

Comment:
We talked about this one on the kubernetes-nmstate PR. Do we need this change?

Reply:
I changed for cluster role not for normal role, I prefer to keep here as we have at knmstate.